### PR TITLE
Aggregating presets by bed objects

### PR DESCRIPTION
**Pull request by sam9111**
Screenshot 2022-03-12 at 7 26 14 PM

Patient presets now aggregate by bed names.

### CODE REVIEW

**Comment by nihal467** [comment]
test approved

File: src/Components/Facility/Consultations/LiveFeed.tsx [hunk 1]
```diff
@@ -89,8 +89,26 @@ const LiveFeed = (props: any) => {
...
+          id: res.bed_object.id,
+          bed_name: res.bed_object.name,
+          preset: [res],
+        });
+      }
+    });
+
+    setBedPresets(aggregatedPresets);
   };
+  console.log(bedPresets);
```
Comment (khavinshankar) [inline src/Components/Facility/Consultations/LiveFeed.tsx:111]:
```suggestion
```

File: src/Components/Facility/Consultations/LiveFeed.tsx [hunk 2]
```diff
@@ -227,6 +245,7 @@ const LiveFeed = (props: any) => {
     { icon: "fa fa-undo", label: "Reset", action: "reset" },
   ];
 
+  console.log(bedPresets);
```
Comment (khavinshankar) [inline src/Components/Facility/Consultations/LiveFeed.tsx:248]:
```suggestion
```

File: src/Components/Facility/Consultations/LiveFeed.tsx [hunk 3]
```diff
@@ -418,55 +437,60 @@ const LiveFeed = (props: any) => {
...
-                          <span className="mx-auto">
+            <div className=" space-y-4 m-2">
+              {showDefaultPresets ? (
+                <div className="grid grid-cols-2 my-auto gap-4 ">
+                  {viewOptions.map((option: any) => (
+                    <div
+                      onClick={() => {
+                        gotoPreset(option.value);
+                      }}
+                    >
```
Comment (khavinshankar) [inline src/Components/Facility/Consultations/LiveFeed.tsx:448]:
```suggestion
                  {viewOptions.map((option: any, i: number) => (
                    <div
                      key={i}
                      onClick={() => {
                        gotoPreset(option.value);
                      }}
                    >
```

File: src/Components/Facility/Consultations/LiveFeed.tsx [hunk 4]
```diff
@@ -418,55 +437,60 @@ const LiveFeed = (props: any) => {
...
+                      <button className="bg-green-100 border border-white rounded-md p-3  text-black  hover:bg-green-500 hover:text-white w-full">
+                        {option.label}
+                      </button>
+                    </div>
+                  ))}
+                </div>
+              ) : (
+                <div className=" ">
+                  {bedPresets.map((bed: any) => (
+                    <div className="space-y-4 m-2">
```
Comment (khavinshankar) [inline src/Components/Facility/Consultations/LiveFeed.tsx:458]:
```suggestion
                  {bedPresets.map((bed: any, i: number) => (
                    <div key={i} className="space-y-4 m-2">
```

**Review by khavinshankar — COMMENTED** [review]
Just some minor changes, otherwise LGTM

**Comment by gigincg** [comment]
Issue resolved in earlier PR

**Pull request closed without merging**